sdk/healthinsights/azure-health-insights-radiologyinsights/README.md

```diff
@@ -0,0 +1,112 @@
+# Azure Cognitive Services Health Insights Radiology Insights client library for Java
+
+
+[Health Insights][health_insights] is an Azure Applied AI Service built with the Azure Cognitive Services Framework, that leverages multiple Cognitive Services, Healthcare API services and other Azure resources.
+
+Radiology Insights is a model that aims to provide quality checks as feedback on errors and inconsistencies (mismatches) and ensures critical findings are identified and communicated using the full context of the report. Follow-up recommendations and clinical findings with measurements (sizes) documented by the radiologist are also identified.
+
+## Getting started
+
+### Prerequisites
+
+- A [Java Development Kit (JDK)][jdk] with version 8 or above
+- [Azure Subscription][azure_subscription]
+- An existing Cognitive Services Health Insights instance.
+
+For more information about creating the resource or how to get the location and sku information see [here][cognitive_resource_cli].
+
+### Adding the package to your product
+
+[//]: # ({x-version-update-start;com.azure:azure-health-insights-radiologyinsights;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-health-insights-radiologyinsights</artifactId>
+    <version>1.0.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Authenticate the client
+
+In order to interact with the Health Insights Clinical Matching service, you'll need to create an instance of the RadiologyInsightsClient class.  You will need an **endpoint** and an **API key** to instantiate a client object.  
```

Comment:
"Health Insights Clinical Matching service" <-- please proofread the doc.

Reply:
Corrected the readme file and made some more changes with references to the services published documentation.